### DISCUSSION TITLE
I want to use only proxy server without vpn setting, is there any way?

### DISCUSSION THREAD

**Discussion by ytoaa** [qdm12/gluetun#931] [Q&A]
I want to use only the proxy server without setting up openvpn and connecting.

**Comment by qdm12** [comment]
Which proxy? The shadowsocks one is available at https://github.com/qdm12/ss-server

The http proxy is not distributed as a separate programs, but there are plenty of other http proxies out there if you need.

**Reply by ytoaa** [reply]
In gluetun, proxy settings were possible after setting up openvpn or wireguard separately. I want to use the proxy right away without going through the previous process.

**Reply by qdm12** [reply]
You can just plug in another container through gluetun to do this. See https://github.com/qdm12/gluetun/wiki/Connect-a-container-to-gluetun

**Reply by ytoaa** [reply]
Sorry, I didn't understand. Can you give me an example?

**Reply by qdm12** [reply]
What are you trying to achieve here? it's a bit unclear so far.

If you just want a proxy server then you can use https://github.com/qdm12/ss-server for example. But again I'm not sure what you're trying to do nor what kind of proxy you're after.
